release.yml

```diff
@@ -0,0 +1,123 @@
+# JIRA Project code where the release issue should be created
+jiraProject:     SB
```

Comment:
Shouldn't this be ENTSB, rather?

Reply:
Yes. For the starter that snowdrop team owns, the JIRA project key should be ENTSBT

Reply:
No. This is the top level JIRA project. This has nothing to do with starters or products.